thirdparty/github.go

```diff
@@ -246,6 +246,9 @@ func githubShouldRetry(caller string, config retryConfig) utility.HTTPRetryFunct
 		url := req.URL.String()
 
 		if err != nil {
+			if errors.Is(err, io.EOF) || errors.Is(err, io.ErrUnexpectedEOF) {
```

Comment:
LGTM! Should we add a comment with context why we retry on it?

Reply:
hmm I wasn't sure if we would need it but a little log never harmed anyone (maybe)